docs/general/fields.md

```diff
@@ -0,0 +1,31 @@
+[Back to Guides](../README.md)
+
+# Fields
+
+If for any reason, you need to restrict the fields returned, you should use `fields` option.
+
+For example, if you have a serializer like this
+
+```ruby
+class UserSerializer < ActiveModel::Serializer
+  attributes :access_token, :first_name, :last_name
+end
+```
+
+and in a specific controller, you want to return `access_token` only, `fields` will help you:
+
+```ruby
+class AnonymousController < ApplicationController
+  def create
+    render json: User.create(activation_state: 'anonymous'), fields: [:access_token], status: 201
+  end
+end
+```
+
+Note that this is only valid for the `json` adapter. For the `json_api` adapter, you would use
```

Comment:
Sorry to add another comment (last one I promise :smile: ) but could you add that it is also valid for the `attributes` adapter (also in the other file) e.g.

```diff
-Note that this is only valid for the `json` adapter.
+Note that this is only valid for the `json` and `attributes` adapters.
```

This is very important as it may be confusing for people using the `attributes` adapter if it is omitted.

And kudos for your quick responsiveness!